modules/components/src/AdvancedFacetView/index.js

```diff
@@ -111,13 +111,15 @@ export default class AdvancedFacetView extends React.Component {
       sqon,
       statsConfig,
       translateSQONValue,
-      onFacetNavigation = () => {},
+      onFacetNavigation,
+      onTermSelected,
+      onClear,
       InputComponent = TextInput,
       ...props
     } = this.props;
     const scrollFacetViewToPath = path => {
       this.facetView.scrollToPath({ path });
-      onFacetNavigation(path);
+      onFacetNavigation?.(path);
```

Comment:
I actually kinda prefer the default props for consistency and also in case it needs to be called at multiple places, we don't forget the check.